src/lib/server/blog/gql.ts

```diff
@@ -18,7 +18,7 @@ export function gql(ghDiscussionCategoryId: string) {
           author {
             login
             url
-            avatarUrl
+            avatarURL
```

Comment:
Since the GitHub GraphQL API does not abide by the same standards as we do, it uses the alternative spelling which must be respected in the GrapQL query and the eventual GraphQL response. 
```suggestion
            avatarUrl
```